ChangeLog

```diff
@@ -1,5 +1,26 @@
 OpenContainers Specifications
 
+Changes with v1.1.1:
+
+	Additions:
+
+	* config: add idmap and ridmap mount options (#1222)
+		* config.md: allow empty mappings for [r]idmap (#1224)
```

Comment:
indent seems broken

Reply:
It was intentional because it is relevant. Should we align?

Reply:
Yes, please

Reply:
Sure! 6f08cb0